.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: python -m pip install --upgrade pip wheel
```

Let's do the wheel installation on a separate line.

```suggestion
      run: |
        python -m pip install --upgrade pip
        pip install wheel
```

---

Done